### PR TITLE
VMware Refactor

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR refactors the VMware page. The platform will have its own section.

## Changed Pages

- [New Index Page for VMware](https://deploy-preview-2540--docs-spectrocloud.netlify.app/clusters/data-center/vmware/)
- [Architecture](https://deploy-preview-2540--docs-spectrocloud.netlify.app/clusters/data-center/vmware/architecture)
- [Create and Manage Clusters](https://deploy-preview-2540--docs-spectrocloud.netlify.app/clusters/data-center/vmware/create-manage-vmware-clusters)
- [Required Permissions](https://deploy-preview-2540--docs-spectrocloud.netlify.app/clusters/data-center/vmware/permissions)

A new page was added to the PCG section:
 - [Add DNS Mapping](https://deploy-preview-2540--docs-spectrocloud.netlify.app/clusters/pcg/manage-pcg/add-dns-mapping)

💻 [Preview URL for Page](https://deploy-preview-2540--docs-spectrocloud.netlify.app/clusters/data-center/vmware/)

## Jira Tickets

🎫 [DOC-448](https://spectrocloud.atlassian.net/browse/DOC-448)

## Backports

Can this PR be backported?

- [x] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-448]: https://spectrocloud.atlassian.net/browse/DOC-448?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/clusters/data-center/vmware/architecture.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  platform and VMware vCenter installed in the private data center.
+
+  The PCG facilitates communication between Palette and your infrastructure environment. The PCG is necessary in
+  environments where Palette does not have direct network access. Many infrastructure environments are placed in a
+  private network that blocks connections originating externally. The PCG connects to Palette, and acts as an endpoint,
+  allowing you to target the environment when deploying clusters in Palette.
+
+  ![vmware_arch_oct_2020.webp](/clusters_vmware_architecture_arch-overview.webp)
+
+You can learn more about the [PCG Architecture](../../pcg/architecture.md) section to learn more about the PCG
```
Comment (addetz) [inline docs/docs-content/clusters/data-center/vmware/architecture.md:31]:
```suggestion
You can learn more in the [PCG Architecture](../../pcg/architecture.md) section.
```

File: docs/docs-content/clusters/data-center/vmware/architecture.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+architecture.
```
Comment (addetz) [inline docs/docs-content/clusters/data-center/vmware/architecture.md:32]:
```suggestion

```

File: docs/docs-content/clusters/data-center/vmware/create-manage-vmware-clusters.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+
+You can deploy Kubernetes clusters on VMware vSphere using Palette. Use the following steps to create and manage VMware
+clusters in Palette.
+
+## Prerequisites
+
+Before you begin, ensure that you have the following prerequisites:
+
+- A VMware vSphere user account with the necessary permissions to create and manage clusters. Refer to the
+  [Require Permissions](./permissions.md) page for details on the required permissions.
```
Comment (addetz) [inline docs/docs-content/clusters/data-center/vmware/create-manage-vmware-clusters.md:18]:
```suggestion
  [Required Permissions](./permissions.md) page for more information.
```

File: docs/docs-content/clusters/data-center/vmware/create-manage-vmware-clusters.md [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+
+- A VMware account registered in Palette. VMware accounts are automatically registered when you deploy a Private Cloud
+  Gateway (PCG) in Palette. Check out the [Deploy a PCG](../../pcg/deploy-pcg/vmware.md) guide to learn how to deploy a
+  PCG.
+
+  :::info
+
+  If you have a self-hosted Palette or VerteX instance, you can use the System PCG that if the instance is deployed in a
```
Comment (addetz) [inline docs/docs-content/clusters/data-center/vmware/create-manage-vmware-clusters.md:26]:
```suggestion
  If you have a self-hosted Palette or VerteX instance, you can use the System PCG instance that is deployed in a
```

File: docs/docs-content/clusters/data-center/vmware/create-manage-vmware-clusters.md [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+   | **Cluster name**  | The name of the cluster.                                                                                                                                                        | Yes      |
+   | **Description**   | A brief description of the cluster.                                                                                                                                             | No       |
+   | **Tags**          | Tags to help you identify the cluster.                                                                                                                                          | No       |
+   | **Cloud Account** | The VMware vSphere account to use for the cluster. If no account is available, ensure you [deployed a PCG](../../pcg/deploy-pcg/vmware.md) into the VMware vSphere environment. | Yes      |
+
+6. Select the cluster profile you want to use for the cluster. Click the **Next** to proceed.
+
+7. Modify any cluster profile layers if needed. Click **Next** to continue.
```
Comment (addetz) [inline docs/docs-content/clusters/data-center/vmware/create-manage-vmware-clusters.md:70]:
```suggestion
7. Modify any cluster profile layers as needed. Click **Next** to continue.
```

File: docs/docs-content/clusters/data-center/vmware/create-manage-vmware-clusters.md [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+   | **Allow Worker Capability**     | Enable this option to workloads to be deployed on control plane nodes.                                                                                                                                                         |
+   | **Additional Labels**           | Additional labels to apply to the control plane nodes.                                                                                                                                                                         |
+   | **Taints**                      | Taints to apply to the control plane nodes. If enabled, an input field is displayed to specify the taint key, value and effect. Check out the [Node Labels and Taints](../../cluster-management/taints.md) page to learn more. |
+
+   #### Cloud Configuration
+
+   | Field Name | Description                                                      |
+   | ---------- | ---------------------------------------------------------------- |
+   | **CPU**    | The number of CPUs to allocate to the control plane nodes.       |
+   | **Memory** | The amount of memory to allocate to the control plane nodes.     |
```
Comment (addetz) [inline docs/docs-content/clusters/data-center/vmware/create-manage-vmware-clusters.md:115]: Should we provide some guidance for the resource amounts in this table?

File: docs/docs-content/clusters/data-center/vmware/create-manage-vmware-clusters.md [hunk 7]
```diff
@@ -0,0 +1,218 @@
...
+You can download the cluster's kubeconfig file to access the cluster using the Kubernetes command-line tool, kubectl.
+Check out the [Access a Cluster](../../cluster-management/palette-webctl.md) guide to learn how to download the
+kubeconfig file.
+
+:::
+
+## Next Steps
+
+Now that you have a Kubernetes cluster deployed, you can start developing and deploying applications to your clusters.
+We recommend you review the day two responsibilities and become familiar with the cluster management tasks. Check out
```
Comment (addetz) [inline docs/docs-content/clusters/data-center/vmware/create-manage-vmware-clusters.md:216]:
```suggestion
We recommend you review the Day-2 responsibilities and become familiar with the cluster management tasks. Check out
```

Comment (addetz) [reply docs/docs-content/clusters/data-center/vmware/create-manage-vmware-clusters.md:216]: This is how this is referred to on this page https://docs.spectrocloud.com/clusters/cluster-management/

File: docs/docs-content/clusters/data-center/vmware/create-manage-vmware-clusters.md [hunk 8]
```diff
@@ -0,0 +1,218 @@
...
+the [Manage Clusters](../../cluster-management/cluster-management.md) documentation to learn more about day two
```
Comment (addetz) [inline docs/docs-content/clusters/data-center/vmware/create-manage-vmware-clusters.md:217]:
```suggestion
the [Manage Clusters](../../cluster-management/cluster-management.md) documentation to learn more about Day-2
```

File: docs/docs-content/clusters/data-center/vmware/vmware.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+sidebar_label: "VMware"
+title: "VMware"
+description: "Learn how to configure VMware to create VMware clusters in Palette."
+hide_table_of_contents: false
+sidebar_position: 30
+tags: ["data center", "vmware"]
+---
+
+Palette supports VMware as a data center provider, enabling you to deploy and manage Kubernetes clusters on VMware
+virtual machines. Palette achieves this by using a [Private Cloud Gateway (PCG)](../../pcg/architecture.md),
```
Comment (addetz) [inline docs/docs-content/clusters/data-center/vmware/vmware.md:11]:
```suggestion
virtual machines. Palette achieves this through a [Private Cloud Gateway (PCG)](../../pcg/architecture.md),
```

File: docs/docs-content/clusters/data-center/vmware/vmware.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+establishing a secure connection from the internal network to the internet-accessible Palette instance and effectively
```
Comment (addetz) [inline docs/docs-content/clusters/data-center/vmware/vmware.md:12]:
```suggestion
which establishes a secure connection from the internal network to the internet-accessible Palette instance and effectively
```

File: docs/docs-content/clusters/data-center/vmware/vmware.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+bypassing NAT gateways and firewalls.
+
+Palette supports VMware as a data center provider. With this, you can deploy and manage Kubernetes clusters on VMware
```
Comment (addetz) [inline docs/docs-content/clusters/data-center/vmware/vmware.md:15]: Repeated paragraph as above?

File: docs/docs-content/clusters/pcg/manage-pcg/add-dns-mapping.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
+---
+sidebar_label: "Add DNS Mapping"
+title: "Add DNS Mapping"
+description: "Learn how to add DNS mapping for a Private Cloud Gateway (PCG) deployed in a VMware vSphere environment."
+hide_table_of_contents: false
+sidebar_position: 15
+tags: ["pcg"]
+---
+
+If your VMware vSphere environment is configures with Dynamic Host Configuration Protocol (DHCP), you can add a Dynamic
```
Comment (addetz) [inline docs/docs-content/clusters/pcg/manage-pcg/add-dns-mapping.md:10]:
```suggestion
If your VMware vSphere environment is configured with Dynamic Host Configuration Protocol (DHCP), you can add a Dynamic
```

File: docs/docs-content/clusters/data-center/vmware/vmware.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
+---
+sidebar_label: "VMware"
+title: "VMware"
+description: "Learn how to configure VMware to create VMware clusters in Palette."
+hide_table_of_contents: false
+sidebar_position: 30
+tags: ["data center", "vmware"]
+---
+
+Palette supports VMware as a data center provider, enabling you to deploy and manage Kubernetes clusters on VMware
```
Comment (addetz) [inline docs/docs-content/clusters/data-center/vmware/vmware.md:10]:
```suggestion
Palette supports VMware as a datacenter provider, enabling you to deploy and manage Kubernetes clusters on VMware
```

Comment (addetz) [reply docs/docs-content/clusters/data-center/vmware/vmware.md:10]: Written as one word elsewhere in this PR.

File: docs/docs-content/clusters/data-center/vmware/create-manage-vmware-clusters.md (hunk 7 above)
Comment (karl-cardenas-coding) [reply docs/docs-content/clusters/data-center/vmware/create-manage-vmware-clusters.md:216]: Excellent catch

File: docs/docs-content/clusters/data-center/vmware/architecture.md [hunk 14]
```diff
@@ -0,0 +1,96 @@
...
+title: "Architecture"
+description: "Learn about the architecture used to support VMware clusters through Palette."
+hide_table_of_contents: false
+sidebar_position: 10
+tags: ["data center", "vmware", "architecture"]
+---
+
+## Overview
+
+The following are some architectural highlights of Kubernetes clusters provisioned by Palette on VMware:
```
[the remark this answers is not in the document]
Comment (karl-cardenas-coding) [reply docs/docs-content/clusters/data-center/vmware/architecture.md:12]: Great feedback

**Pull request merged**